Client/src/components/Pages/Home/Home.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Title from '../../layout/Typography/Title/Title';
+import getLocalData from '../../../api/api';
+
+function Home() {
+  const weatherData = getLocalData();
+
+  const WeatherReport = () => {
+    if (!weatherData) return <p>Loading</p>;
+    return weatherData.data.observations.map(weather => {
+      return (
+        <div key={weather.station}>
+          <div key={weather.station}>Station: {weather.station}</div>
+          <div key={weather.temperature}>Temperature: {weather.temperature}</div>
+          <div key={weather.visibility}>Visibility: {weather.visibility}</div>
+          <div style={{ marginBottom: 20 }} key={weather.winddir}>
```

Comment:
Not sure if you really need key on this element

Reply:
Absolutely right. Only need it on the parent.